client/jetpack-connect/authorize-form.jsx

```diff
@@ -1,67 +0,0 @@
-/** @format */
-/**
- * External dependencies
- */
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import { localize } from 'i18n-calypso';
-
-/**
- * Internal dependencies
- */
-import LoggedInForm from './auth-logged-in-form';
-import LoggedOutForm from './auth-logged-out-form';
-import MainWrapper from './main-wrapper';
-import { authQueryPropTypes } from './utils';
-import { getCurrentUserId } from 'state/current-user/selectors';
-import { recordTracksEvent } from 'state/analytics/actions';
-
-class JetpackConnectAuthorizeForm extends Component {
-	static propTypes = {
-		authQuery: authQueryPropTypes.isRequired,
-
-		// Connected props
-		isLoggedIn: PropTypes.bool.isRequired,
-		recordTracksEvent: PropTypes.func.isRequired,
-	};
-
-	componentWillMount() {
-		this.props.recordTracksEvent( 'calypso_jpc_authorize_form_view' );
-	}
-
-	handleClickHelp = () => {
-		this.props.recordTracksEvent( 'calypso_jpc_help_link_click' );
```

Comment:
Do we lose this event now? Maybe check with support to see if they are using it.

Reply:
No, this was actually [should have moved](https://github.com/Automattic/wp-calypso/pull/20406/files#diff-13f3d900556a872d3fa814d9ef4d18aeR26) in #20406. Instead it was copied. Here it no longer serves, but the event is maintained.